ui/apps/dashboard/src/components/Navigation/Logo.tsx

```diff
@@ -15,9 +14,11 @@ type LogoProps = {
 };
 
 const NavToggle = ({ collapsed, setCollapsed }: LogoProps) => {
-  const toggle = () => {
-    setCollapsed(!collapsed);
-    toggleNav();
+  const toggle = async () => {
+    const toggled = !collapsed;
+    setCollapsed(toggled);
+    typeof window !== 'undefined' &&
```

Comment:
Set cookie using web api instead of next because doing this with next has unpleasant side effects.